configurations/local/logistic.py

```diff
@@ -11,7 +11,7 @@
 import importlib
 import sys
 try:
-    chosen_config=sys.argv[1]
+    chosen_config=sys.argv[1] #NOT OBSOLETE argument!!! needed to import THIS script
```

Comment:
I meant that this line is obsolete, as the variable is not used, but having two command line arguments is necessary